the-event-loop/interval.js

```diff
@@ -0,0 +1,10 @@
+let counter = 3;
+
+const countdown = setInterval(() => {
+  console.log(counter);
+  counter--;
+  if (counter === 0) {
+    clearInterval(countdown);
+    console.log('Blast off!');
+  }
+}, 1000);
```

Comment:
`1` and `"Blast off!"` should output on two separate ticks, not the same one.

Reply:
![interval js](https://user-images.githubusercontent.com/90414477/176080756-78700449-9c17-440e-a246-da5942d0d2b8.gif)